### ISSUE TITLE
Create a CodeFix for Diagnostic Code 7008

### ISSUE THREAD

**Issue by wondersloth** [rehearsal-js/rehearsal-js#368]
When migrating a file, we can add types to classes with the autofix suggested by the language server.

This fix should be similar to the current 7006 implementation.

```
    "Member '{0}' implicitly has an '{1}' type.": {
        "category": "Error",
        "code": 7008
    },
```

**Comment by egorio** [comment]
I think this will fix your issue: https://github.com/rehearsal-js/rehearsal-js/pull/349

**Comment by lynchbomb** [comment]
Def. this should be jsDoc and/or language service plugins

**Comment by lynchbomb** [comment]
This issue will be fixed with #411 (overlap)
